shell/gpu/gpu_surface_gl_skia.h

```diff
@@ -56,23 +57,37 @@ class GPUSurfaceGLSkia : public Surface {
   bool AllowsDrawingWhenGpuDisabled() const override;
 
  private:
-  bool CreateOrUpdateSurfaces(const SkISize& size);
-
-  sk_sp<SkSurface> AcquireRenderSurface(
-      const SkISize& untransformed_size,
-      const SkMatrix& root_surface_transformation);
-
-  bool PresentSurface(const SurfaceFrame& frame, DlCanvas* canvas);
+  struct ViewUnit {
```

Comment:
nitpick: the name of this makes me think that it would be related to units of how a view is measured, rather than a surface with its FBOID and damage rect.

Maybe `SurfaceData` or something?